grails-app/controllers/org/broadinstitute/orsp/api/ClarificationController.groovy

```diff
@@ -23,8 +27,12 @@ class ClarificationController extends AuthenticatedController {
                 render([message: "Error saving comment"])
             }
 
+            IssueReview issueReview = issueReviewService.findByProjectKey(issue.projectKey)
             List<String> toAddresses = new ArrayList<>()
             String fromAddress = (String) getUser()?.emailAddress
+            if (issueReview != null) {
+                toAddresses.add(userService.findUser(issueReview.getEditCreatorName())?.emailAddress)
```

Comment:
This could add a null value to the list. I think you'd want to filter those out.

Reply:
I solved iit this way: 
https://github.com/broadinstitute/orsp-pub/pull/188/files#diff-8940ab54e457fae2a2f136b13aa6c6d0R32